src/main/resources/db/migration/all/20240502000000__create_cas_2_summary_view.sql

```diff
@@ -0,0 +1,22 @@
+-- create application summary view
+DROP VIEW IF EXISTS cas_2_application_summary;
+CREATE OR REPLACE VIEW cas_2_application_summary AS SELECT
+    CAST(a.id AS TEXT),
+    a.crn,
+    a.noms_number,
+    CAST(a.created_by_user_id AS TEXT),
+    nu.name,
+    a.created_at,
+    a.submitted_at,
+    a.hdc_eligibility_date,
+    asu.label,
+    CAST(asu.status_id AS TEXT),
+    a.referring_prison_code
+FROM cas_2_applications a
+LEFT JOIN (SELECT DISTINCT ON (application_id) su.application_id, su.label, su.status_id
+    FROM cas_2_status_updates su
+    ORDER BY su.application_id, su.created_at DESC) as asu
+    ON a.id = asu.application_id
+JOIN nomis_users nu ON nu.id = a.created_by_user_id
+WHERE (a.conditional_release_date IS NULL OR a.conditional_release_date >= current_date)
+ORDER BY a.created_at DESC
```

Comment:
In theory this `ORDER BY` does not need to be there but can't seem to get sorting to work when relying on Pageable to do it.

Reply:
Pagination and sorting issue resolved by now using `getPageableOrAll` since `getPageable` returns null if page is not supplied as query parameter, which means results aren't sorted.